base/broadcast.jl

```diff
@@ -471,7 +471,8 @@ julia> Broadcast.combine_axes(1, 1, 1)
 ```
 """
 @inline combine_axes(A, B...) = broadcast_shape(axes(A), combine_axes(B...))
-combine_axes(A) = axes(A)
+@inline combine_axes(A, B) = broadcast_shape(broadcast_axes(A), broadcast_axes(B))
+@inline combine_axes(A) = axes(A)
```

Comment:
I don't think the `@inline` here matters.

Reply:
yeah, I'll remove it.